tutorials/terraform/data-migration-mysql-mmy/data-migration-mysql-mmy.tf

```
@@ -0,0 +1,126 @@
+# Infrastructure for Yandex Cloud Managed Service for MySQL cluster and Virtual Machine
+#
+# RU: https://cloud.yandex.ru/docs/managed-mysql/tutorials/data-migration
+# EN: https://cloud.yandex.com/en/docs/managed-mysql/tutorials/data-migration
+#
+# Set the following settings:
+locals {
+  # Managed Service for MySQL cluster
+  target_version  = ""   # Set the MySQL version. Мust be the same or higher than the version in the source cluster.
+  target_sql_mode = ""   # Set the MySQL SQL mode. Must be the same as in the source cluster.
+  target_db       = ""   # Set the target cluster database name
+  target_user     = ""   # Set the target cluster username
+  target_pwd      = ""   # Set the target cluster password
+  target_port     = 3306 # Set the target cluster port number for connections from the Internet
+  # (Optional) Virtual Machine
+  vm_image_id   = "" # Set a public image ID from https://cloud.yandex.com/en/docs/compute/operations/images-with-pre-installed-software/get-list
+  vm_public_key = "" # Set a path to SSH public key
+}
+
+# Images with Ubuntu Linux use username `ubuntu` by default.
+variable "vm_user_name" {
+  description = "User name for VM"
+  type        = string
+  default     = "ubuntu"
+}
+
+resource "yandex_vpc_network" "network" {
+  description = "Network for the Managed Service for MySQL cluster and VM"
+  name        = "network"
+}
+
+resource "yandex_vpc_subnet" "subnet-a" {
+  description    = "Subnet in ru-central1-a availability zone"
+  name           = "subnet-a"
+  zone           = "ru-central1-a"
+  network_id     = yandex_vpc_network.network.id
+  v4_cidr_blocks = ["10.1.0.0/24"]
+}
+
+resource "yandex_vpc_security_group" "security-group" {
+  description = "Security group for the Managed Service for MySQL and VM"
+  network_id  = yandex_vpc_network.network.id
+
+  ingress {
+    protocol       = "TCP"
+    description    = "Allow connections to MySQL from the Internet"
```

```suggestion
    description    = "Allow connections to Managed Service for MySQL cluster from the Internet"
```

Или просто `Allow connections to cluster from the Internet`. Сейчас слова пропущены.